gradle-spaghetti-typescript-plugin/src/main/java/com/prezi/spaghetti/typescript/gradle/internal/DefinitionFileComparator.java

```diff
@@ -0,0 +1,29 @@
+package com.prezi.spaghetti.typescript.gradle.internal;
+
+import java.io.File;
+import com.prezi.typescript.gradle.SerializableFileComparator;
+
+public class DefinitionFileComparator extends SerializableFileComparator {
+    private static final long serialVersionUID = 1L;
+
+    public static DefinitionFileComparator INSTANCE = new DefinitionFileComparator();
+
+    private DefinitionFileComparator() {
+    }
+
+    public int compare(File a, File b) {
+        if (a.getPath().endsWith(".module.ts")) {
```

Comment:
If both of them (`a` and `b`) end with `module.ts`, this will be inconsistent:
* `compare(a, b)` will be 1, `compare(b, a)` will be 1 too